tests/unit/preprocessor/_volume/test_volume.py

```diff
@@ -7,7 +7,8 @@
 from cf_units import Unit
 
 import tests
-from esmvalcore.preprocessor._volume import (volume_statistics,
+from esmvalcore.preprocessor._volume import (axis_statistics,
+                                             volume_statistics,
                                              depth_integration,
                                              extract_trajectory,
                                              extract_transect,
```

Comment:
Should we sort this list alphabetically? (All of it, of course, but github doesn't allow me to extend the comment context so far.)